app/views/devise/confirmations/new.html.erb

```diff
@@ -1,15 +1,15 @@
-<h2>Resend confirmation instructions</h2>
+<h1 class="h4 text-gray-900 mb-4">Resend confirmation instructions</h1>
```

Comment:
Internationalization for 'Resend confirmation instructions'

Reply:
done